### ISSUE TITLE
Labels order if you set shuffle=True in the dataloader

### ISSUE THREAD

**Issue by cnyanhao** [yukimasano/self-label#21]
Hi, thanks for your great paper. I've noticed that you set `shuffle=True` in your dataloader `t_loader` and let `train_loader` and `pseudo_loader` point to the same object `t_loader`. I'm wondering in this way whether the optimised labels are in the right order when they are used to train the model since you will shuffle the data again.

**Comment by cnyanhao** [comment]
Alright, I got it. Yes, the data will be shuffled but it doesn't matter. Since you used `selected` to track the dataloader sampling and labels fetching, the images and labels will always be in right correspondence.
